cmd/common/st/transfer_backups.go

```diff
@@ -0,0 +1,46 @@
+package st
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal/storagetools/transfer"
+)
+
+const backupsShortDescription = "Moves all backups from one storage to another"
+
+// backupsCmd represents the backups command
+var backupsCmd = &cobra.Command{
+	Use:   "backups [backup_name] --source='source_storage' [--target='target_storage']",
+	Short: backupsShortDescription,
+	Args:  cobra.RangeArgs(0, 1),
+	Run: func(_ *cobra.Command, args []string) {
+		var fileLister transfer.FileLister
+		if len(args) == 0 {
+			fileLister = transfer.NewAllBackupsFileLister(transferOverwrite, transferMaxFiles, adjustMax(transferMaxBackups))
+		} else {
+			fileLister = transfer.NewSingleBackupFileLister(args[0], transferOverwrite, transferMaxFiles)
+		}
+
+		cfg := &transfer.HandlerConfig{
+			FailOnFirstErr:           transferFailFast,
+			Concurrency:              transferConcurrency,
+			AppearanceChecks:         transferAppearanceChecks,
+			AppearanceChecksInterval: transferAppearanceChecksInterval,
+		}
+
+		handler, err := transfer.NewHandler(transferSourceStorage, targetStorage, fileLister, cfg)
+		tracelog.ErrorLogger.FatalOnError(err)
+
+		err = handler.Handle()
+		tracelog.ErrorLogger.FatalOnError(err)
+	},
+}
+
+var transferMaxBackups int
+
+func init() {
+	backupsCmd.Flags().IntVar(&transferMaxBackups, "max-backups", -1,
```

Comment:
Actually, regarding the `-1` logic, is there any reason to introduce it? We can check if flag is set via `.IsSet()` method.

Reply:
Hmm, didn't know about that. Yes, I can do it so. `--max-files` flag is currently behaving the same way

Reply:
I just realised, that it's impossible to hide the `(default <N>)` note in the command help description, even if we call `.IsSet()` on our own. Given that we need the infinite number here (actually, max int value), it will look either weird (very long integer number) or incorrect (for example, the default value in help is `0`, but in fact, it is max int).

Reply:
Just made the default value = `math.MaxInt`